controllers/news.controller.js

```diff
@@ -0,0 +1,16 @@
+const fetch = require('node-fetch');
+
+exports.getNews = async (req, res, next) => {
+  const source = `https://www.gamespot.com/api/games/?api_key=${process.env.GAMESPOTAPIKEY}&format=json`
+
+  try {
+    const response = await fetch(source);
+    const results = await response.json();
```

Comment:
When possible avoid general names like `results` or `data`. Sometimes we have to but in other places like this one we have opportunity to name it for what is is.